rfc/helm-import.md

```diff
@@ -0,0 +1,515 @@
+# Helm Import 
+
+* **Original Author(s):** @vinayak-kukreja
+* **Tracking Issue:** https://github.com/cdk8s-team/cdk8s/issues/1271
+* **API Bar Raiser:** @iliapolo
+
+Users are now able to import helm charts into their cdk8s app with `cdk8s import` command.
+
+## Working Backwards
+
+### README
+
+You can add a helm chart to your cdk8s application by running:
+
+```
+cdk8s import helm:https://airflow.apache.org/airflow@1.8.0
+```
+
+Here, `airflow` is the helm chart and `1.8.0` is the chart version.
+
+Then you can use this chart within your application:
+
+```typescript
+import { Construct } from 'constructs';
+import { App, Chart, ChartProps } from 'cdk8s';
+import { Airflow } from './imports/airflow';
+
+export class MyChart extends Chart {
+  constructor(scope: Construct, id: string, props: ChartProps = { }) {
+    super(scope, id, props);
+
+    new Airflow(this, 'airflow', {
+      values: {
+        fullnameOverride: 'Foo',
+      }
+    });
+  }
+}
+
+const app = new App();
+new MyChart(app, 'typescript-demo');
+app.synth();
+```
+
+> **Note:**
+>
+> * You would need `Helm` to be installed on your machine for using this feature. 
+> * We assume that you are authenticated to access the helm chart being mentioned in the url.
```

Comment:
```suggestion
> * For accessing private helm repositories, you must be authenticated to the repository in a way that the `helm pull` command recognizes.
```

Reply:
We should also add this disclaimer to our current Helm documentation :)

Reply:
Good callout, on it. :)

Reply:
PR: https://github.com/cdk8s-team/cdk8s/pull/1411